predictoor/pdr-faq.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Predictoor FAQ
+description: Frequently Asked Questions about Predictoor
+---
+
+## Predictoor FAQ
+
+### Getting Started
+
+<details>
+
+<summary>How can users begin experimenting with Predictoor?</summary>
+
+You can start by playing with [predictoor.ai](https://, then click "run bots" and go through [Predictoor](https://github.com/oceanprotocol/pdr-backend/blob/main/READMEs/predictoor.md) or [Trader](https://github.com/oceanprotocol/pdr-backend/blob/main/READMEs/trader.md) README. This [blog](https://blog.oceanprotocol.com/meet-predictoor-accountable-accurate-prediction-feeds-8b104d26a5d9#048b) "How to Earn" section has details.  
+
+</details>
+
+### How is Predictoor different
+<details>
+
+<summary>How does Predictoor compare to prediction markets? Like Predictoor, they use $ for accountability.</summary>
+
+Prediction markets are for one-off predictions like “will Biden win in 2024?” whereas Predictoor-style prediction feeds are for a continuous stream of predictions.
+
+</details>
+
+<details>
+
+<summary>How does Predictoor compare to Chainlink? Like Predictoor, it has price feeds on-chain.</summary>
+
+Chainlink provides current prices on-chain, and Predictoor makes predictions for prices in the future.
+
+</details>
+
+<details>
+
+<summary>How does Predictoor compare to Numerai? Like Predictoor, it crowd-sources predictions, related to trading.</summary>
+
+They are different:  
+- Numerai takes predictions, then aggregates them and uses them to trade directly. It operates as a hedge fund.  
+- Whereas Predictoor takes predictions, then aggregates them and sells the aggregate. It operates as data feeds.  
+
+Secondary differences:  
+- Numerai focus is tradfi trading, as a hedge fund.  
+- Whereas Predictoor is pure datafeeds, and applies to any vertical. Its first use case is defi trading.  
+- Currently Numerai is mostly centralized.  
+- Whereas Predictoor is decentralized. Predictoor tech may be useful to Numerai to help decentralize  
+
+</details>
+
+<details>
+
+<summary>How does Predictoor compare to Numerai Signals? People supplying to Numerai Signals don’t get paid out from trading (like the main Numerai product).</summary>
+
+In Numerai Signals, payout is a function of the submitter’s “originality” compared to other signals. These signals are used as inputs for prediction models in Numerai main product.  
+
+Whereas in Predictoor, payout is a function of the the accuracy of the submitter’s prediction, compared to the true value. These signals are predictions, directly.
+
+</details>
+
+### For Predictoors
+
+<details>
+
+<summary>What happens if your prediction is good or bad?</summary> 
+
+If the prediction is wrong, your staked amount on that prediction is going to be slashed and divided between people who submitted the right predictions.
```

Comment:
Change `divided` -> `distributed`. Why: more clear

Change 'between` -> `among`. Why: proper grammar. "Between" is for two parties, "among" is for more.